examples/AspNet/Global.asax.cs

```diff
@@ -66,6 +68,37 @@ protected void Application_Start()
 
             this.tracerProvider = builder.Build();
 
+            // Metrics
+            // Note: Tracerprovider is needed for metrics to work
+            // https://github.com/open-telemetry/opentelemetry-dotnet/issues/2994
+
+            var meterBuilder = Sdk.CreateMeterProviderBuilder()
+                 .AddAspNetInstrumentation()
+                 .AddHttpClientInstrumentation();
```

Comment:
HttpClient for .NET FW don't produce metrics as of today, so its best to remove this line to avoid confusion. We can add it when httpclient starts having some metrics.